src/utility.cpp

```diff
@@ -80,6 +89,26 @@ data_chunk new_seed(size_t bit_length)
     return seed;
 }
 
+data_chunk new_electrum_seed(size_t bit_length)
+{
+    static const auto now = high_resolution_clock::now();
```

Comment:
I could see moving most of this method into libbitcoin/src/utility/random.cpp similar to new_seed